idp/dex/web/templates/webauthn_verify.html

```
@@ -0,0 +1,12 @@
+{{ template "header.html" . }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)